docs/concepts/what-is-a-data-converter.md

```diff
@@ -83,7 +93,9 @@ In codec implementations, we recommended running the function (whether it be com
 - [`ZlibCodec`](https://github.com/temporalio/sdk-go/blob/706516c7077ba2e9b40304aeddbed47e25b2a68f/converter/codec.go#L77-L105) in the Go SDK
 - [Encryption Data Converter](https://github.com/temporalio/samples-go/blob/15be864c80d4d983ebb8a8fbd3fa5263bcef6930/encryption/data_converter.go#L100-L126) in Go's encryption sample
 
-### Encryption
+To view data that's been converted by a codec in the Web UI and tctl, use a [Codec Server](/server/security#codec-server).
```

Comment:
I would mention that the codec can encode payloads too which can be used both in the UI and `tctl`.

Reply:
It's talking about payloads—here's the context:

https://github.com/temporalio/documentation/blob/2ba2552c4093cf1039f4c2333755b7be45cf1033/docs/concepts/what-is-a-data-converter.md#payload-codec

Reply:
This is the Payload Codec section of the Data Converter concept page. I added this line to link to the codec server page because people using a payload codec may want to see either data in web ui/tctl.

My last comment was just saying that I think "data" is understood to be a payload.